CMakeLists.txt

```diff
@@ -32,9 +32,9 @@ endif()
 #
 #  NOTE: This is used due to legacy requirements instead of:
 #    `install(FILE <license files> COMPONENT Documentation DESTINATION share/aproj/licenses)`
-configure_file(LICENSE ${CMAKE_CURRENT_BINARY_DIR}/licenses/LICENSE COPYONLY)
+configure_file(LICENSE licenses/LICENSE COPYONLY)
```

Comment:
I may change this back in [cdt/#112](https://github.com/AntelopeIO/cdt/pull/114) if submoduling won't work with this

Reply:
Hmm, you might need to change it to something different. The replacement line and original should be functionally the same.